src/TemplateLibrary/index.js

```diff
@@ -120,12 +119,11 @@ const TemplateLibraryComponent = (props) => {
    * @return {*} the react component
    */
   const filtered = filterTemplates(props.templates);
-  const libraryProps = props.libraryProps || Object.create(null);
 
   return (
-      <TemplatesWrapper>
+      <TemplatesWrapper className='ciceroUI'>
```

Comment:
Shouldn't this class name be more specific?

Reply:
Keeping @DianaLease comment in mind, so I need to add specific class names?

Reply:
I think @DianaLease is correct, so this could be left.